gordo_components/model/factories/feedforward_autoencoder.py

```diff
@@ -46,6 +47,7 @@ def feedforward_model(
     decoding_dim = dec_dim or [64, 128, 256]
     encoding_func = enc_func or ["tanh", "tanh", "tanh"]
     decoding_func = dec_func or ["tanh", "tanh", "tanh"]
+    out_func = out_func or "linear"
```

Comment:
No, this can be a concrete default argument like `     out_func: str = "linear", ` in the parameter. That is cleaner. The only reason we are not doing that for the encoding/decoding func is because they are lists.

Reply:
there is a separate issue to align the code in the feedforward_autoencoder with the lstm_autoencoder, one such change would be the one you commented on. I thought that for this pull request i leave the format as is and then change everything in another pull request linked to the mentioned issue.